dev/com.ibm.ws.microprofile.faulttolerance.2.1.cdi/src/com/ibm/ws/microprofile/faulttolerance21/cdi/config/impl/FallbackConfig21Impl.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.faulttolerance21.cdi.config.impl;
+
+import java.lang.reflect.Method;
+
+import javax.enterprise.inject.spi.BeanManager;
+import javax.interceptor.InvocationContext;
+
+import org.eclipse.microprofile.faulttolerance.Fallback;
+
+import com.ibm.ws.microprofile.faulttolerance.cdi.config.impl.FallbackConfigImpl;
+import com.ibm.ws.microprofile.faulttolerance.spi.FallbackPolicy;
+
+/**
+ *
+ */
+public class FallbackConfig21Impl extends FallbackConfigImpl {
```

Comment:
In general, either write something in the javadoc comment or remove it completely